tests/testthat/test-GLV.R

```diff
@@ -0,0 +1,20 @@
+usethis::use_r("simulateGLV")
```

Comment:
What is the requirement for creating an R file for a test? As far as I can tell it is never used in the test, is it?

Reply:
No, it is not used in the test. It only refers to the its R script, but can be deleted. I will delete it, thank you

Reply:
I already deleted these in the other PR as they caused trouble